src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,174 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.NoSuchElementException;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
     @Override
     public void add(T value) {
+        if (isEmpty()) {
+            Node<T> node = new Node<>(null, value, null);
+            head = node;
+            tail = node;
```

Comment:
you have the same line on the 20th, let's put it after if else, for this purpose you will need to rewrite your logic